pysatSpaceWeather/instruments/sw_dst.py

```diff
@@ -28,9 +28,8 @@
 
 import datetime as dt
 import ftplib
-import os
 import numpy as np
-
+import os
```

Comment:
isn't `os` part of core?

Reply:
Unless they're separated into blocks they should be listed alphabetically.  And they weren't separated into blocks.